src/main/java/com/quizter/service/UserService.java

```diff
@@ -136,7 +136,7 @@ public User getUserPrincipal() {
     }
 
     public StudentDto findStudentByEmail(String email) {
-        return groupMapper.toStudentDto(userRepository.findUserByEmail(email));
+        return groupMapper.toStudentDto(userRepository.findUserByEmail(email).orElseThrow());
```

Comment:
put in orElseThrow scopes ResourseNotFoundException

Reply:
Ok